src/components/LeftNavigation.js

```diff
@@ -5,7 +5,7 @@ const LeftNavigation = () => {
   return (
     <nav className="navigation">
       <NavLink to={"/new"} className="navigation__link">Create</NavLink>
-      <NavLink to={"/jobs"} className="navigation__link">Active Jobs</NavLink>
+      <NavLink to={"/jobs"} className="navigation__link">Jobs</NavLink>
```

Comment:
Maybe add a section specifically for Running Jobs? (running and/or finished but un-finalized? How does app behave if user navigates away while job is running, i.e. to start another ingest? Is it possible that a job would be left unfinalized?)

Reply:
So I think a separation of active/inactive jobs will be addressed in the next iteration where active and inactive jobs will be stored differently. Trying to get this to a finished stopping point to come back to the next wave of enhancements.

If the user navigates away while a job is running, the calls from the ingest steps still get made. It's taken care in a higher level component.

Reply:
Ok no worries